releasenotes/notes/39493.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: release-notes/v2
+
+kind: feature
+
+area: installation
+
+issue:
+  - 39493
```

Comment:
this should be a number reference to an issue on https://github.com/istio/istio/issues, not the PR#.  If there is not an associated github issue, can be left blank

Reply:
Hi @GregHanson,
https://github.com/istio/istio/issues/39493 should work though. 

For GitHub, issues and pull requests are the same and having the pull request linked would add contextual value.